modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,71 @@
+== Kiali architecture
+
+Kiali Server (back end):: This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali Server does not need storage. When deploying the Server to a cluster, configurations are set in ConfigMaps and secrets.
+
+Kiali console (front end):: The Kiali console is a web application. 	The Kiali Server serves the Kiali console, which then queries back to the Kiali Server for data to present it to the user.
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+Red Hat Service Mesh (Istio):: Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and will not work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the {product-title} cluster API.
+
+Prometheus:: A dedicated Prometheus instance optional. When Istio telemetry is enabled, metrics data are stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetry. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali's features will not work without Prometheus.
+
+{ocp-product-title} API:: Kiali uses {ocp-product-title} API to fetch and resolve service mesh configurations. Kiali queries the cluster API to retrieve, for example, definitions for namespaces, services, deployments, pods, and other entities. Kiali also makes queries to resolve relationships between the different cluster entities. The cluster API is also queried to retrieve Istio configurations like virtual services, destination rules, route rules, gateways, quotas, and so on.
+
+Tracing:: Tracing is optional, but when you install {DTProductName} and Kiali is configured, the Kiali console includes a tab to display distributed tracing data, and tracing integration on the graph itself. Note that tracing data will not be available if you disable Istio’s distributed tracing feature. Also note that the user must have access to the namespace where the user needs to see tracing data.
+
+Grafana:: Grafana is optional. When available, the metrics pages of Kiali display links to direct the user to the same metric in Grafana. Note that Grafana is not supported as part of {ocp-product-title} or {SMProduct}.
+
+[id="kiali-features_{context}"]
+== Kiali features
+
+The Kiali console is integrated with {SMProduct} and provides the following capabilities:
+
+Health:: Quickly identify issues with applications, services, or workloads.
+
+Topology:: Visualize how your applications, services, or workloads communicate via the Kiali graph.
```

Comment:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v): Use this term, which means by way of, to refer to routing data, such as in “Messages are routed via SMTP”.

```suggestion
Topology:: Visualize how your applications, services, or workloads communicate through (?) the Kiali graph.
```